editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -216,6 +217,7 @@
 
              ;; From BundleGenericHandler
              "build-server" build-server-url
+             "build-server-headers" build-server-headers
```

Comment:
Will this work? We're only supplying the argument once to Bob, and it will be a newline-separated string.

Reply:
No, this will no longer work.... I'm not sure how to change this to call bob with multiple --build-server-header?

Reply:
The editor does not call Bob's main, instead, it uses this bob args map to set options using `com.dynamo.bob.Project::setOption` method. We should either change Bob to get build server headers from that options map or make the editor handle `"build-server-headers"` in a bespoke way when configuring the Project instance in `editor.pipeline.bob/bob-build!` fn.

Reply:
Ok, I see. Then maybe we should add so that bob can also accept `setOption(String, List<String>)`?

Reply:
Actually, I think there is a simpler solution that should be done differently — don't make server headers bob args at all. `generic-bundle-bob-args` is a misleading name because it suggests this is a CLI args for bob, while in reality, it's a map of options — not args! — for the Project class. Build server headers are not an option to the Project class, it's a list of strings that should be set with the `addBuildServerHeader` method, and it shouldn't be here in the options at all. Instead, server headers should be a separate argument to `bob-build!` passed all the way from `editor.app-view` ns (i.e. pass it to calls to `editor.disk/async-bob-build!`, then forward it to `editor.pipeline.bob/bob-build!`, then call `addBuildServerHeader` on `bob-project`)

Reply:
Thanks Vlad. This is the correct solution.